.github/workflows/docker-build.yaml

```diff
@@ -29,18 +29,25 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-      - name: Build and push
+      - name: Build
         uses: docker/build-push-action@v3
         with:
           context: .
-          push: ${{ github.ref == 'refs/heads/master' }}
+          outputs: type=docker,dest=/tmp/docker.tar
           tags: ${{ secrets.DOCKERHUB_USERNAME }}/bancho.py:latest
           cache-from: type=gha
           cache-to: type=gha,mode=max
-          outputs: type=docker,dest=/tmp/docker.tar
 
       - name: Upload artifact
         uses: actions/upload-artifact@v3
         with:
           name: docker
           path: /tmp/docker.tar
+
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v3
+        if: ${{ github.ref == 'refs/heads/master' }}
+        run: |
+          docker image load -i /tmp/docker.tar

```

Comment:
is this necessary? or will the image already be available from the build step @cmyui

Reply:
changed